src/index.ts

```diff
@@ -883,6 +883,19 @@ export class Bigtable {
     }
   }
 
+  /**
+   * Terminate grpc channels and close all the clients.
+   */
+  close(): Promise<void[]> {
+    const combined = Object.keys(this.api).map(clientType =>
+      this.api[clientType].close()
+    );
+    if (combined.length === 0) {
```

Comment:
Just a small nitpick here. I did a test and it looks like `Promise.resolve` does the same thing as `Promise.all` with an empty array. So that means that these last 4 lines can just be `return Promise.all(combined);` unless there was some specific reason you added the if statement.

![image](https://user-images.githubusercontent.com/8935272/149795148-7f1d65d2-2452-44e1-80f3-d8494e5fabad.png)